support/utils/command.go

```diff
@@ -75,10 +76,24 @@ func FindOrInstallGoBasedProvider(pPath, provider, module, version string) (stri
 	return "", fmt.Errorf("%s not available even after installation", provider)
 }
 
+// RunCommand run command and returns an *exec.Proc with information about the executed process.
 func RunCommand(command string) *exec.Proc {
 	return commandRunner.RunProc(command)
 }
 
+// RunCommandWithSeperatedOutput run command and returns the result as stdout, stderr and err.
+func RunCommandWithSeperatedOutput(command string) (stdout, stderr string, err error) {
```

Comment:
@heylongdacoder I thought about this change further.
I have a suggestion. Since this method is splitting the  command output streams, maybe it should accept the stream target as io streams instead of returning strings like so:

```go
func RunCommandWithSeperatedOutput(command string, stdout, stderr io.Writer) (err error) {
}
```
That way, the streams can be processed as needed (i.e. piped to a file, byte buffers, etc)